proto/Data/Telemetry/RpcResponseTime.proto

```diff
@@ -1,10 +1,149 @@
 syntax = "proto3";
 package POGOProtos.Data.Telemetry;
 
-import "POGOProtos/Networking/Requests/RequestType.proto";
-
 message RpcResponseTime {
-	.POGOProtos.Networking.Requests.RequestType rpc_id = 1;
+	Method rpc_id = 1;
 	int32 count_call = 2;
 	float average_response_time = 3;
+
+	enum Method {
```

Comment:
I'm not confortable with this, why not use the RequestType we already have?

Reply:
I do not know, I prefer to put the original method, but if I see that it's wrong to put the requestype

Reply:
It's not wrong, just use the good request type. we don't want to places to update requesttype